src/node_ref.rs

```diff
@@ -1758,16 +1782,18 @@ where
     ///     .collect();
     /// assert_eq!(bfs_leaves, [5, 8, 9, 10, 11]);
     /// ```
-    fn leaves<T>(&'a self) -> impl Iterator<Item = &'a V::Item>
+    fn leaves<'t, T>(&self) -> impl Iterator<Item = &'a V::Item> + 't
     where
-        T: Traverser<OverData>,
+        T: Traverser<OverData> + 't,
+        <T as TraverserCore>::Storage<V>: 't,
```

Comment:
Couldnt this line be omitted as (to my knowledge) it is already inherited by the above `T: /*...*/ + 't`?